### ISSUE TITLE
object has no attribute 'app_id'

### ISSUE THREAD

**Issue by moimikey** [jaapz/cast#1]
thoughts?

```
Traceback (most recent call last):
  File "/usr/local/bin/cast", line 8, in <module>
    load_entry_point('cast==0.1', 'console_scripts', 'cast')()
  File "/Library/Python/2.7/site-packages/cast.py", line 82, in main
    cast = pychromecast.PyChromecast(CHROMECAST_HOST)
  File "/Library/Python/2.7/site-packages/pychromecast/__init__.py", line 109, in __init__
    self.refresh()
  File "/Library/Python/2.7/site-packages/pychromecast/__init__.py", line 147, in refresh
    cur_app.app_id != app.app_id)
AttributeError: 'NoneType' object has no attribute 'app_id'
```

```
mshertzberg@forge ~/.config/cast $ cat config.ini
[cast]
chromecast_ip=10.0.1.2
sleep_time=0.3
```

```
mshertzberg@forge ~/.config/cast $ cast --version
cast 0.1
```

**Comment by jaapz** [comment]
Looks like a bug in pychromecast, ping @balloob.
@moimikey can you pdb into it and check if cur_app or app is None?

**Comment by jaapz** [comment]
It's not that the chromecast does not exist, because then it should give a `pychromecast.error.ConnectionError`. I will try to reproduce this later this week.

**Comment by moimikey** [comment]
I think this was just stupidity on my part. I didn't take into consideration as to what would happen if the stream wasn't RAMP, which was the case and probably explains the app_id error. I was attempting to stream a non RAMP via http://movies.foamsnet.com/url/. It works as it should though using YouTube for example.

```
mshertzberg@forge ~/Git/cast (master*) $ cast status
 ▶ Simon Draws: Rabbits by None from None via YouTube, 0:0 of 3:1
```

---

I unfortunately don't have experience with PDB, but for the sake of what I was looking at:

```
>>> import time
>>> import pychromecast as pc
>>> cast = pc.PyChromecast()
>>> print cast.app
None
>>> print cast.cur_app
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'PyChromecast' object has no attribute 'cur_app'
>>> print cast.device
DeviceStatus(friendly_name='CHROMEO', model_name='Eureka Dongle', manufacturer='Google Inc.', api_version=(1, 0))
>>>
```

**Comment by balloob** [comment]
This is a known bug in v0.3.2 and has been fixed in balloob/pychromecast@541ad6de80645655393fe5ee18f5502ca0ee82e4 

I'll push out a new version to pip tonight.

**Comment by moimikey** [comment]
:+1:  closing
